site/website/versioned_docs/version-v1.0.0/3_manage_nifi/1_manage_clusters/1_deploy_cluster/3_expose_cluster/2_istio_service_mesh.md

```diff
@@ -3,3 +3,135 @@ id: 2_istio_service_mesh
 title: Istio service mesh
 sidebar_label: Istio service mesh
 ---
+
+The purpose of this section is to explain how to expose your NiFi cluster using Istio on Kubernetes.
+
+## Istio configuration for HTTP
+
+To access to the NiFi cluster from the external world, it is needed to configure a Gateway and a VirtualService on Istio.
+
+The following example show how to define a Gateway that will be able to intercept all requests for a specific domain host on HTTP port 80.
+
+```yaml
+apiVersion: networking.istio.io/v1beta1
+kind: Gateway
+metadata:
+  name: nifi-gateway
+spec:
+  selector:
+    istio: ingressgateway
+  servers:
+  - port:
+      number: 80
+      name: http
+      protocol: HTTP
+    hosts:
+    - nifi.my-domain.com
+```
+
+In combination, we need to define also a VirtualService that redirect all requests itercepted by the Gateway to a specific service. 
```

Comment:
In fact,  I was suggesting to put \`VirtualService\` and \`Gateway\` to have a nice formatting